frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareTitleDetailsPage.tsx

```diff
@@ -133,6 +145,13 @@ const SoftwareTitleDetailsPage = ({
               name={softwareTitle.name}
               source={softwareTitle.source}
             />
+            {showPackageCard && (
+              <SoftwarePackageCard
+                softwarePackage={createMockSoftwarePackage()}
```

Comment:
could we put in the call to the software package API and have this returned from your mock API call instead of directly from the mock data?